apps/dashboard/src/app/nebula-app/(app)/components/CustomChat/CustomChatButton.tsx

```
@@ -0,0 +1,93 @@
+  isFloating: boolean;
+  pageType: "chain" | "contract" | "support";
+  label: string;
+  client: ThirdwebClient;
+  customApiParams: any;
+  examplePrompts: ExamplePrompt[];
+  authToken: string | undefined;
+  requireLogin?: boolean;
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [hasBeenOpened, setHasBeenOpened] = useState(false);
+  const [isDismissed, setIsDismissed] = useState(false);
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend setIsDismissed with an underscore.


(lint/correctness/noUnusedVariables)